### PR TITLE
Added AbortReason logging

### PR DESCRIPTION
**Pull request by kosecki123**

### CODE REVIEW

File: src/Enum/AbortReason.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export enum AbortReason {
+  WasCancelled, //0
+  AlreadyCalled, //1
+  BeforeCallWindow, //2
+  AfterCallWindow, //3
+  ReservedForClaimer, //4
+  InsufficientGas, //5
+  TooLowGasPrice,
```
Comment (lsaether) [inline src/Enum/AbortReason.ts]: Did you mean to skip a comment on this line?

File: test/unit/UnitTestActions.ts [hunk 2]
```diff
@@ -95,4 +98,154 @@ describe('Actions Helpers Unit Tests', () => {
...
+
+      assert.equal(ExecuteStatus.ABORTED_TOO_LOW_GAS_PRICE, executeStatus);
+    });
+
+    it('returns ExecuteStatus.ABORTED_UNKNOWN when unknown reason appeared', () => {
+      const receipt = {
+        logs: [
+          {
+            topics: [ABORTED_EVENT],
+            data: '0x0000000000000000000000000000000000000000000000000000000000000008'
```
Comment (lsaether) [inline test/unit/UnitTestActions.ts]: Did you mean to skip `0x0000000000000000000000000000000000000000000000000000000000000007` in the tests? It threw me off for a second.0x0000000000000000000000000000000000000000000000000000000000000008

**Review by lsaether — APPROVED** [review]

File: src/Enum/AbortReason.ts (hunk 1 above)
Comment (kosecki123) [reply src/Enum/AbortReason.ts]: I copy pasted this enum from Solidity EAC :)

File: test/unit/UnitTestActions.ts (hunk 2 above)
Comment (kosecki123) [reply test/unit/UnitTestActions.ts]: The AbortReason are from 0 to 6, anyway above is unknown from client code pov.

File: src/Enum/AbortReason.ts (hunk 1 above)
Comment (lsaether) [reply src/Enum/AbortReason.ts]: You pushed the comment for index 6 a line down.

**Pull request merged**